src/Makevars.in

```diff
@@ -1,14 +1,16 @@
 include sources.mk
+include vendor/cigraph/vendor/glpk/sources-glpk.mk
 
 PKG_CFLAGS=$(C_VISIBILITY)
 PKG_CXXFLAGS=$(CXX_VISIBILITY)
 PKG_FFLAGS=$(F_VISIBILITY)
 
-PKG_CPPFLAGS=-DUSING_R -I. -Ivendor -Ivendor/cigraph/src -Ivendor/cigraph/include -Ivendor/cigraph/vendor @cflags@ \
+# FIXME: Make INTERNAL_GMP depending on availability of gmp.h
```

Comment:
Using an external GMP provides no real benefit at the moment, other than not having to deal with CRAN's complaints about compiler warnings.

Attention: Do not patch `mini-gmp.c` to eliminate warnings! GMP is too easy to break accidentally.